api/grpcserver/v2alpha1/layer_test.go

```diff
@@ -124,7 +135,7 @@ func TestLayerStreamService_Stream(t *testing.T) {
 
 		var streamed []layers.Layer
 		for i := start + 0; i < start+n; i++ {
-			layer, err := generateLayer(db, types.LayerID(i), true)
+			layer, err := generateLayer(db, types.LayerID(i), true, true)
```

Comment:
Perhaps generateLayer could take _options_ instead of 2 bools. For example, I find this more readable and less error-prone:
```suggestion
			layer, err := generateLayer(db, types.LayerID(i), genProcessed(true), genWithBlock(true))
```